release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -158,9 +158,356 @@ For more information, see xref:../applications/creating_applications/creating-ap
+|Supported
+|Supported
+
+|Horizontal pod autoscaling
+|Supported
+|Supported
+
+|IBM Secure Execution
+|Unsupported
+|Supported
+
+|{ibmpowerProductName} Virtual Server Block CSI Driver Operator (Technology Preview)
+|Supported
+|Unsupported
+
+|Installer-provisioned Infrastructure Enablement for {ibmpowerProductName} Virtual Server (Technology Preview)
+|Supported
+|Unsupported
+
+|Installing on a single node
+|Supported
+|Supported
+
+|IPv6
+|Supported
+|Supported
+
+|Monitoring for user-defined projects
+|Supported
+|Supported
+
+|Multi-architecture compute nodes
+|Supported
+|Supported
+
+|Multipathing
+|Supported
+|Supported
+
+|Network-Bound Disk Encryption - External Tang Server
+|Supported
+|Supported
+
+|Non--volatile memory express drives (NVMe)
+|Supported
+|Unsupported
+
+|oc-mirror plugin
+|Supported
+|Supported
+
+|OpenShift CLI (`oc`) plugins
+|Supported
+|Supported
+
+|Operator API
+|Supported
+|Supported
+
+|OpenShift Virtualization
+|Unsupported
+|Unsupported
+
+|OVN-Kubernetes, including IPsec encryption
+|Supported
+|Supported
+
+|PodDisruptionBudget
+|Supported
+|Supported
+
+|Precision Time Protocol (PTP) hardware
+|Unsupported
+|Unsupported
+
+|{openshift-local-productname}
+|Unsupported
+|Unsupported
+
+|Scheduler profiles
+|Supported
+|Supported
+
+|Stream Control Transmission Protocol (SCTP)
+|Supported
+|Supported
+
+|Support for multiple network interfaces
+|Supported
+|Supported
+
+|Three-node cluster support
+|Supported
+|Supported
+
+|Topology Manager
+|Supported
+|Unsupported
+
+|z/VM Emulated FBA devices on SCSI disks
+|Unsupported
+|Supported
+
+|4K FCP block device
+|Supported
+|Supported
+|====
+
+.Persistent storage options
+[cols="2,1,1",options="header"]
+|====
+|Feature |{ibmpowerProductName} |{ibmzProductName} and {linuxoneProductName}
+|Persistent storage using iSCSI
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using local volumes (LSO)
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using hostPath
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using Fibre Channel
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using Raw Block
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+
+|Persistent storage using EDEV/FBA
+|Supported ^[1]^
+|Supported ^[1]^,^[2]^
+|====
+[.small]
+--
+1. Persistent shared storage must be provisioned by using either {rh-storage-first} or other supported storage protocols.
+2. Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA.
+--
+
+.Operators
+[cols="2,1,1",options="header"]
+|====
+|Feature |{ibmpowerProductName} |{ibmzProductName} and {linuxoneProductName}
+
+|Cluster Logging Operator
+|Supported
+|Supported
+
+|Cluster Resource Override Operator
+|Supported
+|Supported
+
+|Compliance Operator
+|Supported
+|Supported
+
+|File Integrity Operator
+|Supported
+|Supported
+
+|HyperShift Operator
+|Technology Preview
+|Technology Preview
```

Comment:
```suggestion
// |HyperShift Operator
// |Technology Preview
// |Technology Preview
```
Needs to be added through CM after 4.14 GA see discussion in this [thread](https://redhat-internal.slack.com/archives/CFFJUNP6C/p1694770713321609).

Reply:
Updated the PR.